src/components/catalogNoResultsDeck/CatalogNoResultsDeck.jsx

```diff
@@ -37,14 +39,22 @@ function CatalogNoResultsDeck({
 
   useEffect(() => {
     const defaultCoursesRefinements = { enterprise_catalog_query_titles: selectedCatalog, content_type: contentType };
+    if (contentType === CONTENT_TYPE_COURSE) {
+      if (courseType !== null) {
+        defaultCoursesRefinements.course_type = [courseType];
+      } else {
+        // if a course type is not specified, default to edx course content
+        defaultCoursesRefinements.course_type = EDX_COURSES_COURSE_TYPES;
+      }
+    }
```

Comment:
nit: We can shorten the code here, but it's not necessarily a win in terms of readability, so will leave it to your discretion.
```suggestion
       // if a course type is not specified, default to edx course content 
      defaultCoursesRefinements.course_type = courseType !== null ? [courseType] : EDX_COURSES_COURSE_TYPES;
```